course3/homework-string/build.bat

```diff
@@ -0,0 +1,22 @@
+@echo off
```

Comment:
I started out that way, but I couldn't get it to work without needing a .bat to run cmake correctly... in the end it was getting overly complex for something very simple. So for now I went with a simple .bat, to be revisited when the projects get more complex.

I had something like this:
```
project(test)

cmake_minimum_required(VERSION 3.5.2)
set(CMAKE_CXX_COMPILER C:/MinGW/bin/g++.exe)
set(CMAKE_C_COMPILER C:/MinGW/bin/gcc.exe)
set(CMAKE_LINKER C:/MinGW/bin/ld.exe)
set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14 -Wall -Werror -pedantic -g")


add_executable(${PROJECT_NAME}
test.cpp
)
```

But then I had to run
```
cmake ../ -G "MSYS Makefiles"
```

to get it to build the make file before running make, which I kept messing up.